src/south/ProtocolHandler.class.js

```diff
@@ -134,7 +134,7 @@ class ProtocolHandler {
     this.logger.info(`Data source ${dataSourceId} started with protocol ${protocol}`)
   }
 
-  async onScanImplementation(scanMode) {
+  async git ad(scanMode) {
```

Comment:
???

Reply:
Sorry, I didn't realized that IntelliJ was the active window instead of Terminal when I started to type `git add`. I fixed it.